04/cipher_caesar.py

```diff
@@ -0,0 +1,41 @@
+diacritics = ['á', 'é', 'í', 'ó', 'ú', 'ý', 'č', 'ď', 'ě', 'ň', 'ř', 'š', 'ť', 'ž', 'ů']
+ASCII_numbers = [49, 50, 51, 52, 53, 54, 55, 56, 57, 58]
+
+key = input('key ')
+for znak in key:
+    znak_ASCII = ord(znak)
+    if znak_ASCII not in ASCII_numbers:
+        key = int(input('Zadávej jen celá, kladná čísla. Příklad: 2, 5, 15 \nKey: '))
+
+key = int(key)
+
+if key <= 0:
+    key = int(input('Key můžou být jen celá, kladná čísla. Příklad: 2, 5, 15 \nKey: '))
+
+plain_text = input('plain text ')
+for character in plain_text:
+    if character in diacritics:
+        plain_text = input('Bez čísel a diakritiky. Příklad: les, pes, jedna \nPlain text: ')
+    elif character.isnumeric():
```

Comment:
Prečo nie čísla?

Reply:
musela bych na to vyladit kód, pokud by se to mělo posouvat o klíč, tak by bylo potřeba zahrnout rozsah v ASCII tabulce. Mám to nastavené jen na znaky abecedy.  Určitě by byla možnost šifrovat i čísla, případně je nechat jen vypsat. Jen jsem při tvorbě uvažovala, že tam není číslo.

Reply:
Čiste teoreticky by si mala skúsiť zaistiť presun čo najviac znakov, t.j čísla, písmená (malé, veľké) a tak isto nejak spracovať ak tam užívateľ vloží znaky `., *+` a pod. Tu zrovna nechceme toho užívateľa limitovať a aj ak by tam zadal písmenká s diakritikou tak by som to spracovala nejakým spôsobom (buď neprevádzať alebo z nich diakritiku odobrať :) ).